test-app/tsconfig.json

```
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
```

This should be:

```suggestion
    "allowSyntheticDefaultImports": false,
    "strict": true,
```

(See [the discussion of **Module interop**](https://www.semver-ts.org/#module-interop) in the SemVer for TS spec for why we want to set `allowSyntheticDefaultImports` to `false` here!)

---

…have we not *released* `ember-cli-typescript` since I made those blueprint updates? I'll check.

---

We had not, so I have now done so: ember-cli-typescript v5.1 includes the updated blueprints!